testcases.py

```diff
@@ -135,7 +135,7 @@ def _is_valid_keylog(self, filename) -> bool:
         if not os.path.isfile(filename):
             return False
         with open(filename, "r") as file:
-            if not re.search(r"^SERVER_HANDSHAKE_TRAFFIC_SECRET", file.read()):
+            if not re.search(r"^SERVER_HANDSHAKE_TRAFFIC_SECRET", file.read(), re.M):
```

Comment:
`re.MULTILINE` is also valid, isn't it? That would be easier to read without looking it up in the docs.